src/test/java/seedu/momentum/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,149 @@
+    }
+
+    @Test
+    public void execute_deadlineAscending_sortedInDeadlineAscendingOrder() {
+        expectedModel.orderFilteredProjectList(SortType.DEADLINE, true);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "deadline, ", "ascending");
+        assertCommandSuccess(DEADLINE_ASCENDING, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
+
+    @Test
+    public void execute_deadlineDescending_sortedInDeadlineDescendingOrder() {
+        expectedModel.orderFilteredProjectList(SortType.DEADLINE, false);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "deadline, ", "descending");
+        assertCommandSuccess(DEADLINE_DESCENDING, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
+
+    @Test
+    public void execute_createdDateAscending_sortedInCreatedDateAscendingOrder() {
+        expectedModel.orderFilteredProjectList(SortType.CREATED, true);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "created date, ", "ascending");
+        assertCommandSuccess(CREATED_DATE_ASCENDING, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
+
+    @Test
+    public void execute_createdDateDescending_sortedInCreatedDateDescendingOrder() {
+        expectedModel.orderFilteredProjectList(SortType.CREATED, false);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "created date, ", "descending");
+        assertCommandSuccess(CREATED_DATE_DESCENDING, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
+
+    @Test
+    public void execute_nullSortTypeAscendingNonDefault_sortedInCurrentSortTypeAscendingOrder() {
+        model.orderFilteredProjectList(SortType.DEADLINE, false);
+        expectedModel.orderFilteredProjectList(SortType.DEADLINE, true);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "", "ascending");
+        assertCommandSuccess(NULL_SORT_TYPE_ASCENDING_NON_DEFAULT, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
+
+    @Test
+    public void execute_nullSortTypeDescendingNonDefault_sortedInCurrentSortTypeDscendingOrder() {
+        model.orderFilteredProjectList(SortType.CREATED, true);
+        expectedModel.orderFilteredProjectList(SortType.CREATED, false);
+        String expectedMessage = String.format(MESSAGE_SORT_SUCCESS, "", "descending");
+        assertCommandSuccess(NULL_SORT_TYPE_DESCENDING_NON_DEFAULT, model, expectedMessage, expectedModel);
+        assertEquals(model.getFilteredProjectList(), expectedModel.getFilteredProjectList());
+    }
```

Comment:
I think you can extract the common string variables that you have reused in this section and make them private static variables. For example, `ascending`, `descending`, `alphabetical`, `deadline`, `created date`

Reply:
Fixed!